pipelines/Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent any
+
+    options {
+        ansiColor('xterm')
+        disableConcurrentBuilds()  // Don't run 2 concurrent builds at the same time
+        buildDiscarder logRotator(numToKeepStr: '8')  // Maximum number of builds to save
+    }
+
+    parameters {
+        //Extended info param for builds list
+        extendedChoice name: 'ACTION',
+                       value: 'tract_config_logback, bapi_config_logback',
```

Comment:
It must be real ansible tags, like "common_ntp_install", "common_banner_configure", etc